docs/source/reference/video_classification.rst

```diff
@@ -0,0 +1,223 @@
+
+.. _video_classification:
+
+####################
+Video Classification
+####################
+
+********
+The task
+********
+
+Typically, Video Classification is used to identify video clips containing a single object.
+The task predicts which ‘class’ the video clip most likely belongs to with a degree of certainty.
+A class is a label that describes what is in the video clip, such as ‘car’, ‘house’, ‘cat’ etc.
+For example, we can train the video classifier task on video clips with human actions
+and it will learn to predict the probability that a video contains a certain human action.
+
+Lightning Flash :class:`~flash.video.VideoClassifier` and :class:`~flash.video.VideoClassificationData`
+relies on `PyTorchVideo <https://pytorchvideo.readthedocs.io/en/latest/index.html>`_ internally.
+
+You can use any models from `PyTorchVideo Model Zoo <https://pytorchvideo.readthedocs.io/en/latest/model_zoo.html>`_
+with the :class:`~flash.video.VideoClassifier`.
+
+------
+
+**********
+Finetuning
+**********
+
+The :class:`~flash.video.VideoClassifier` provides several pre-trained model.
+
+.. code-block:: python
+
+    import sys
+
+    import torch
+    from torch.utils.data import SequentialSampler
+
+    import flash
+    from flash.data.utils import download_data
+    from flash.video import VideoClassificationData, VideoClassifier
+    import kornia.augmentation as K
+    from pytorchvideo.transforms import ApplyTransformToKey, RandomShortSideScale, UniformTemporalSubsample
+    from torchvision.transforms import Compose, RandomCrop, RandomHorizontalFlip
+
+    # 1. Download a video dataset
+    download_data("PATH_OR_URL_TO_DATA")
+
+    # 2. [Optional] Specify transforms to be used during training.
+    train_transform = {
+        "post_tensor_transform": Compose([
```

Comment:
Why do you call this "post_tensor"?

Reply:
Because there is pre_tensor, to_tensor, post_tensor transforms.
It enables to fully separate the transformation logic and give more control.